package.json

```diff
@@ -14,12 +14,15 @@
     "dist"
   ],
   "scripts": {
-    "lint": "yarn run prettier:check && yarn run lint:eslint",
     "lint:eslint": "eslint . --ext .js,.ts",
+    "lint": "yarn run prettier:check && yarn run lint:eslint",
     "prettier:check": "prettier --check \"./**/*.{js,ts,md,json}\" --loglevel silent",
-    "prettier:write": "prettier --write \"./**/*.{js,ts,md,json}\""
+    "prettier:write": "prettier --write \"./**/*.{js,ts,md,json}\"",
+    "test": "jest",
+    "tsc": "tsc --project tsconfig.json"
```

Comment:
Should this one be "compile" or "build" instead?

Reply:
Could be, but that one is mainly for linting the code. I'll create a proper build script in next PR.